core/templates/components/entity-creation-services/story-creation.service.ts

```diff
@@ -39,12 +40,8 @@ angular.module('oppia').factory('StoryCreationService', [
         if (storyCreationInProgress) {
           return;
         }
-        $uibModal.open({
-          templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-            '/pages/topic-editor-page/modal-templates/' +
-            'create-new-story-modal.template.html'),
-          backdrop: 'static',
-          controller: 'CreateNewStoryModalController'
+        return NgbModal.open(CreateNewStoryModalComponent, {
+          backdrop: true
         }).result.then(function(newlyCreatedStory) {
```

Comment:
I'm not sure about this, but is `backdrop: 'static'` the same as `backdrop: true`?

Reply:
No, It was written by mistake.
Updated now